response_operations_ui/controllers/uaa_controller.py

```diff
@@ -200,3 +200,33 @@ def create_user_account(email, password, user_name, first_name, last_name):
             )
 
     return errors
+
+
+def update_user_account(payload):
```

Comment:
Can you put a docstring here describing what the function does and what the payload should be?  With context, it's clear the payload is whatever you got from `GET /users/<id>` then tweaked.  But someone who has never touched uaa wouldn't know that.